assets/prometheus-user-workload/prometheus.yaml

```diff
@@ -262,3 +265,10 @@ spec:
         memory: 100Mi
     version: 0.35.0
   version: 2.52.0
+  volumes:
+  - configMap:
+      items:
+      - key: ca-bundle.crt
```

Comment:
I don't know if kube makes sure that the key value isn't empty before starting the Pod and if the value can be empty during a transient/temporary period...
(but even in that case, maybe, the Pod will converge)

Reply:
no Kubernetes wouldn't check that but yes it should converge eventually.